nwbib-cdv-2019-jury/index.html

```diff
@@ -0,0 +1,282 @@
+				<a href="https://nwbib.de/BT000063880"><img src="img/lobid-link.png" style="border: none; box-shadow: none;" width=700 /></a>
+			</section>
+
+			<section>
+				<section>
+					<h3>JSON: via Klick oder <a href="http://lobid.org/resources/BT000063880.json">.json</a>
+					<a href="https://lobid.org/resources/BT000063880"><img src="img/lobid-einzeltreffer.png" style="border: none; box-shadow: none;" width=700 /></a>
+				</section>
+				<section>
+				<h3>Angabe des Ortsbezugs im JSON</h3>
+				<pre><code>{
+  "spatial":[
+      {
+          "focus":{
+              "id":"http://www.wikidata.org/entity/Q1250595",
+              "geo":{
+                  "lat":51.5189,
+                  "lon":7.32528
+              },
+              "type":[
+                  "http://www.wikidata.org/entity/Q253019"
+              ]
+          },
+          "id":"https://nwbib.de/spatial#Q1250595",
+          "type":[
+              "Concept"
+          ],
+          "label":"Bövinghausen",
+          "source":{
+              "id":"https://nwbib.de/spatial",
+              "label":"Raumsystematik der Nordrhein-Westfälischen Bibliographie"
+          }
+      }
+  ]
+}</code></pre>
+				</section>
+			</section>
+
+			<section>
+				<h1>Wie es zur Idee des Quiz' kam</h1>
+				<p><span class="fragment">unkonkrete Kartenvisualisierungsidee bei der <a href="http:/slides.lobid.org/nwbib-cdv/#/16">Vorstellung des Datensets</a></span></p>
+
+				<p><span class="fragment">konkrete Idee konnte auf dem Anfangstreffen nicht gefunden werden </span></p>
+				<p><span class="fragment">ist mir dann Abends auf der Heimreise im Zug eingefallen</a></span></p>
+				<p><span class="fragment">Dank des CdV Teams (@cdvwest) reagierte @_hanwong <a href="https://twitter.com/_hanwong/status/1186571496451444736"</a>und verwies auf seinen nachnutzbaren Code</span></p>
+			</section>
+
+			<section>
+				<h3>Einstiegsseite</h3>
+				<a href="https://lobid.org/download/tmp/nrw-quiz/"><img src="img/Einstiegsseite.png" style="border: none; box-shadow: none; height: 50%; width: 50%" /></a>
+			</section>
+
+			<section>
+				<h3>Model View Controller</h3>
+				<p><span class="fragment">jedes Level hat seinen eigenen geojson-Datensatz</a></span></p>
+				<p><span class="fragment">ein neues Geo-Datenset muss mit nur einer Zeile konfiguriert werden</a></span></p>
+				<p><span class="fragment">ansonsten bleibt alles gleich (Logik, Rendern)</a></span></p>
+				<p><span class="fragment">=&gt; leicht erweiterbar</a></span></p>
+			</section>
+
+			<section>
+				<h3>Clou</h3>
+				<p><span class="fragment">Die Quiz-Sets (Bilder,Geodaten ...) sind automatisiert erzeugt!</a></span></p>
+			</section>
+
+			<section>
+				<h3>Kartenoverlay</h3>
+				<p><span class="fragment">um NRW prominenter in die OSM einzubauen: <a href="http://mapwarper.net/maps/42916">topologische NRW Karte aus Wikicommons mittels Map Warper alignen und Tileset erstellen lassen</a></span></p>
```

Comment:
Das ist glaube ich nicht so richtig versändlich. Meinst du "um NRW prominent in OSM hervorzuheben"?